style.css

```diff
@@ -37,81 +46,138 @@
     --color-gdscript: var(--nord-frost-3);
 
     --icon-padding: 0.5em;
+
+    --spacer: 4em;
+
+    --font-size-sm: .8rem;
+    --font-size-med: 1rem;
+    --font-size-lg: 1.2rem;
+    --font-size-xl: 1.5rem;
 }
 
 /* General */
-.no-style {
-    list-style: none;
-    padding: 0;
+html {
```

Comment:
when using CSS, go largest scope to smallest scope. It makes debugging easier. I updated. accordingly